source/bin/tvm.rs

```diff
@@ -114,14 +115,24 @@ pub fn process_options(arguments: Vec<String>) {
                     },
 
                     _ => {
-                        println!("Invalid option “{}”.\n", argument);
+                        println!("Invalid option \"{}\".\n", argument);
```

Comment:
I do like the (language dependent) correct ones more too, but imagine someone was using a host platform that does not support them. Rust nearly eliminates this problem compiler-side, but imagine the problems of transferring source code or blobs containing platform independent data.

Generally using those quotes can introduce problems to a minority of users, which have no way of getting around that.